Core/EventHandlers.lua

```diff
@@ -2071,6 +2077,25 @@ end
 
 local IsQuestFlaggedCompleted = C_QuestLog.IsQuestFlaggedCompleted
 
+function Rarity:grabAura()
+	local rotrBuff = 407063 -- Rocks on the Rocks buff
+	local unit = "player"
+	Rarity:Debug("Checking aura on the player character.")
+	local hasOpenedCOMGToday = IsQuestFlaggedCompleted(7325)
+	if not hasOpenedCOMGToday then
+		AuraUtil.ForEachAura(unit, "HELPFUL", nil, function(_, _, _, _, _, _, _, _, _, spellId)
+			if spellId == rotrBuff then
+				addAttemptForItem("Brul", "pets")
+			else
+				Rarity:Debug("Aura Not found on " .. unit)
```

Comment:
Inconsistent spelling aside, it should probably mention what aura was expected to be found on what unit (especially since there will be other auras once the primal storm PR is merged):

```lua
Rarity:Debug(format("Required aura %s not found on unit %s", L["Rocks on the Rocks"], unit))
```

Not sure what format the other PR will use, but it should probably be consistent.

Reply:
yeah or it can be scrapped along with some of the other debug messages, only really needed for testing purposes